.github/workflows/ucentralgw-dev-deployment.yaml

```diff
@@ -126,17 +138,6 @@ jobs:
           export OWPROVUI_VERSION=${{ matrix.owprovui_version }}
           export OWANALYTICS_VERSION=${{ matrix.owanalytics_version }}
           export OWSUB_VERSION=${{ matrix.owsub_version }}
-          export VALUES_FILE_LOCATION=values.openwifi-qa.yaml,values.openwifi-qa.external-db.yaml
-          export RTTY_TOKEN=${{ secrets.RTTY_TOKEN }}
-          export OWGW_AUTH_USERNAME=${{ secrets.UCENTRALGW_AUTH_USERNAME }}
-          export OWGW_AUTH_PASSWORD=${{ secrets.UCENTRALGW_AUTH_PASSWORD }}
-          export OWFMS_S3_SECRET=${{ secrets.UCENTRALFMS_S3_SECRET }}
-          export OWFMS_S3_KEY=${{ secrets.UCENTRALFMS_S3_KEY }}
-          export CERT_LOCATION=cert.pem
-          export KEY_LOCATION=key.pem
-          export OWSEC_NEW_PASSWORD=${{ secrets.OWSEC_NEW_PASSWORD }}
-          export MAILER_USERNAME=${{ secrets.MAILER_USERNAME }}
-          export MAILER_PASSWORD=${{ secrets.MAILER_PASSWORD }}
           ./deploy.sh
         else
           export NAMESPACE=${{ matrix.namespace }}
```

Comment:
Isn't the variable also duplicate at this point?

Reply:
Yep, thanks for noticing!